packages/atomic/src/components/commerce/facets/facet-number-input/atomic-commerce-facet-number-input.tsx

```diff
@@ -57,18 +57,11 @@ export class FacetNumberInput {
   }
 
   private get minimumInputValue() {
-    const {domain} = this.facet.state;
-    const ensureValue = (num?: number) =>
-      isUndefined(num) ? Number.MIN_SAFE_INTEGER : num;
-    return Math.max(ensureValue(this.start), ensureValue(domain?.min));
+    return isUndefined(this.start) ? Number.MIN_SAFE_INTEGER : this.start;
```

Comment:
Negative price is weird.... 
Could check if `this.field === ec_price || ec_promo_price` and if so, set minimum to 0 instead of `Number.MIN_SAFE_INTEGER`